.env_SAMPLE

```diff
@@ -48,8 +48,10 @@ export ALLOW_ADMIN_URL=True
 # See https://github.com/kennethreitz/dj-database-url for URL formatting.
 #########################################################################
 
-export DATABASE_URL=sqlite:///db.sqlite3
-#export DATABASE_URL=postgres://cfpb@localhost/cfgov
+# export DATABASE_URL=sqlite:///db.sqlite3
+export DATABASE_URL=postgres://cfpb@localhost/cfgov
```

Comment:
Unifying this environment variable is problematic because if not using Docker, we want the host value here to be `localhost` if running Postgres using something like Homebrew, but in Docker it needs to refer to the Postgres container, which is `postgres`.